### ISSUE TITLE
Translations not picked up for Text component

### ISSUE THREAD

**Issue by hiralvasa** [adobe/aem-core-forms-components#1330]
Hi Team,

I have created a form and dictionary entries are present for different locale(en,fr,de).
After adding the form on a page with lang fr, a few of the translations are not picked up for "/apps/core/fd/components/form/text/v1/text" component on dispatcher or when refreshed the page with "view as published".

Without adding "wcmmode=disabled" to the url, all the translations work as expected.
When refreshed the form with "wcmmode=disabled&afAcceptLang=fr-fr", all the translations work well.

Can you please let me know if I am doing something wrong or this is an issue/any workarounds?

AEM version 6.5.17
core components version 1.1.27-wcmB004

Thanks in advance!

**Comment by rismehta** [comment]
> I have created a form and dictionary entries are present for different locale(en,fr,de).
After adding the form on a page with lang fr, a few of the translations are not picked up for "/apps/core/fd/components/form/text/v1/text" component on dispatcher or when refreshed the page with "view as published".

How are you adding the form to the page? Are you using the Form Embed component? If so, are you embedding the form in iframe or non-iframe mode?

**Comment by hiralvasa** [comment]
Hi @rismehta,

Form gets embedded as non-iframe mode.

Trying to get the translation working on local : 
Text on form : 
<img width="1204" alt="image" src="https://github.com/user-attachments/assets/588b8f0c-af77-47ad-8fa6-b443b92d276f">

Dict value for the Text : 
<img width="782" alt="image" src="https://github.com/user-attachments/assets/22689a03-8825-4af0-8393-1ae9082ee339">

This is just for one component - /apps/core/fd/components/form/text/v1/text. Rest all the components are translated successfully. Infact, this is the same behaviour on publish as well. Apart from the Text component, all other components works fine.

**Comment by hiralvasa** [comment]
Sure @rismehta,

But is there a workaround with SP17 that I can accommodate? Since, upgrading to SP20 will need to go through a long process and might take some time.

**Comment by rismehta** [comment]
> But is there a workaround with SP17 that I can accommodate? Since, upgrading to SP20 will need to go through a long process and might take some time.

@dmaurya929 Can you check if workaround exists for SP 17 to unblock the current use-case

**Comment by hiralvasa** [comment]
@rismehta @dmaurya929 ,

This is only happening for the Text Component (/apps/core/fd/components/form/text/v1/text).
All other core components, like email, textinput, dropdown, checkbox, etc are getting translated properly.

Kindly help me with a workaround for the above scenario. Thanks.

**Comment by rismehta** [comment]
> @rismehta @dmaurya929 ,
> 
> This is only happening for the Text Component (/apps/core/fd/components/form/text/v1/text). All other core components, like email, textinput, dropdown, checkbox, etc are getting translated properly.
> 
> Kindly help me with a workaround for the above scenario. Thanks.

@hiralvasa We have identified and fixed the issue, and the changes have been merged into the corresponding branch. You can review the details here: https://github.com/adobe/aem-core-forms-components/pull/1333.

To unblock your use-case, I have created a new release that includes these changes, https://github.com/adobe/aem-core-forms-components/releases/tag/core-forms-components-reactor-1.1.27-wcmB014

**Comment by rismehta** [comment]
Closing this issue
